build/scripts/build_mono.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+mkdir -p ../../Binaries/mono_build
+cd ../../Binaries/mono_build
+
+sudo apt-get install git autoconf libtool automake build-essential mono-devel gettext cmake
```

Comment:
Good that we're installing git on a machine that already cloned this repo to begin with. ;-)

Reply:
Where would you expect such documentation to exist?

Reply:
This is copied from http://www.mono-project.com/docs/compiling-mono/linux/#debian-based-distributions

> already cloned this repo

Not necessarily. One could have downloaded this repo via tarball or zip.